plugins/index.js

```diff
@@ -203,3 +207,16 @@ export async function loadPluginsIfNecessary() {
         }
     });
 }
+
+// Closes RHS if its showing this plugin.
```

Comment:
For consistency with other comments in this file, suggest:
```suggestion
 // hideRHSPlugin closes the RHS if currently showing this plugin.
```